infra/cifuzz/fuzz_target.py

```diff
@@ -30,6 +34,15 @@
 
 LIBFUZZER_OPTIONS = '-seed=1337 -len_control=0'
 
+# Location of google cloud storage for latest OSS-Fuzz builds.
+GCS_BASE_URL = 'https://storage.googleapis.com/'
+
+# Location of cluster fuzz builds on GCS.
+CLUSTER_FUZZ_BUILDS = 'clusterfuzz-builds'
```

Comment:
nit: `clusterfuzz` is a one word

this is also coming from another CL, right?

Reply:
Fixed.